### ISSUE TITLE
Output labels/ROIs in Cellpose2.0 GUI

### ISSUE THREAD

**Issue by Eddymorphling** [MouseLand/cellpose#519]
Hi folks,
Thank you for Cellpose2.0! I have been using it segment cells from large 3D datasets.  This has been very helpful in training to create my own models using the GUI.  

The output labels in the current Cellpose GUI I believe is set to `uint16` which creates a maximum of 65535 3D labels/ROIs. Here is what I noticed - every time I run Cellpose on 3D datasets using the GUI - the total number of predicted ROIs is always 65535 cells. I think this is due to the fact that the output label is set to `uint16` and I believe that my 3D datasets have more than 65535 cells in them. Is there a way to in the Cellpose GUI to create like a binarized output (0-1) instead of creating 16 bit labels labels using the GUI? I hope this makes sense. Thanks again!

**Comment by carsen-stringer** [comment]
cellpose 2.0 should be using uint32 in this case. can you update to the latest version? which 3D segmentation option are you using?

**Comment by Eddymorphling** [comment]
Hi @carsen-stringer I am using Cellpose 2.0.5. I use my own model (created with the cellpose GUI) and use the `stitch_threshold `function in the GUI when performing 3D segmentation. Should I update it to the latest github version?

**Comment by MathieuBo** [comment]
Morning!

I observed the same bug. Running cellpose from a notebook, the output `masks` variable is `uint16`. 
This is segmentation of a large 3D dataset (15000px x 15000px x 68 z planes). Labels are continuous in planes 0 to 42 and then label 0 appear which I guess comes from uint16 overflow. 

I am running cellpose 2.0.5.
The model I used is a custom own, trained with GUI and then run in a notebook. I use it with 3D==False and use the stitch_threshold parameter. 

Thanks!
